_posts/2024-09-27-learning-a-programming-language-with-generative-ai.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: Learning a programming language with generative ai
+author: andandersson
+excerpt: >
+  Experiences from learning a programing language with generatve ai.
+tags:
+ - Generative ai
+ - C#
+ - .net
+---
+
+# Background
+
+Generative artificial intelligence, or GenAI, is one of the biggest hypes in the software industry. With the introduction of applications like ChatGPT, AI-algorithms can produce text, generate images and even motion pictures and music. Due to the trends, there have been many discussions how GenAI may affect software development. How can it be utilized,for example for coding, learning new tools and frameworks, code reviewing? 
+
+![A genAI-created picture of a robot coding](/img/2024-09-27-learning-a-programming-language-with-generative-ai/coding-robot.png)
+
+In the GenAI-twin project, we wanted to find out these things. For the period of March-September 2024, we started to develop a full stack application in C# with .NET and React with the help of GenAI. In that way, we were able to get a picture of how GenAI can affect software development.
```

Comment:
"In the GenAI-twin project" -> Do readers have an idea what "the GenAI-twin project is? Should it be "In Solita's GenAI Twin project?

Reply:
corrected